pinot-core/src/test/java/org/apache/pinot/queries/MultiValueRawQueriesTest.java

```diff
@@ -49,11 +49,7 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNotNull;
-import static org.testng.Assert.assertNull;
-import static org.testng.Assert.assertThrows;
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.*;
```

Comment:
import Assert instead of star import?

Reply:
We usually import `Assert.*` in test to simplify the code